src/gluonts/dataset/parallelized_loader.py

```diff
@@ -202,7 +202,7 @@ def batchify(
         referenced by identical key are reduced using the stack function"""
     return {
         key: stack(
-            data=[item[key] for item in data],
+            data=[item.get(key, f"Key: {key} not found!") for item in data],
```

Comment:
Note that this will cause the string `f"Key: {key} not found!"` to be returned in place of `item[key]` whenever `key` is not found. Is this intended?

Reply:
Yes, basically it would fail here before if the key was item in the first time series in the dataset but not in later ones.  So we returned this message here in case.  We can update that this though if you like.

Reply:
I think in that case this will cause `stack` to fail because some part of `data` are arrays while some are strings (error messages). Maybe we can leave it as it is, and let `item[key]` raise the error?

Reply:
Yes, I will revert this.